reactive/jujushell.py

```diff
@@ -78,7 +78,10 @@ def install_jujushell():
 def install_termserver():
     hookenv.status_set('maintenance', 'fetching termserver')
     try:
-        jujushell.save_resource('termserver', jujushell.termserver_path())
+        jujushell.save_resource(
+            'termserver', jujushell.termserver_path())
+        jujushell.save_resource(
+            'limited-termserver', jujushell.termserver_path(limited=True))
```

Comment:
`save_resource` pulls the resource down from the charmstore, right? Maybe we want to only save the resource we need per the config, and grab the other one if the config changes?